### ISSUE TITLE
picaro combate bug

### ISSUE THREAD

**Issue by lexintor** [RoDNet/Bugtracker-Infinium-4.0.6a#788]
Aluvion de acero no funciona

**Comment by ghost** [comment]
http://es.wowhead.com/spell=65956

**Comment by lexintor** [comment]
?

**Comment by ghost** [comment]
He puesto el link de la spell de wowhead, ya que no es que lo hayas documentado... mucho xD Es para facilitarles el trabajo ;)

**Comment by lexintor** [comment]
Supongo que tendran internet para mirarlo

**Comment by Keough** [comment]
-.-! hay que aprender a reportar un bug xD lee el foro hay una guía de como hacerlo...

**Comment by ghost** [comment]
Exacto, hay que saber reportar un bug y no decir tal cosa no funciona y que se busquen la vida ellos. Claro que tienen internet, pero si les facilitamos el trabajo de búsqueda muchísimo mejor ;)

**Comment by Haruyuuki** [comment]
a ver la habilidad no realiza su función de daño a un enemigo adicional ni tampoco aumenta el consumo de energía de las demás habilidades aunque se activa perfectamente pero no realiza sus funciones
